hexrdgui/masking/mask_manager.py

```diff
@@ -113,6 +124,7 @@ def data(self, values):
         self.update_masked_arrays()
 
     def update_masked_arrays(self, view=ViewType.raw):
+        self.last_view_mode = view
```

Comment:
Since the threshold masks don't actually use the view mode, maybe `get_masked_arrays` should be overloaded for it, and skip the view mode check?

Reply:
Sounds good!